src/solvers.jl

```diff
@@ -11,11 +11,11 @@ import StochasticDiffEq: SDEProblem
 import DiffEqJump: JumpProblem
 
 funcindex!(list, key, f, vals...) = list[key] = f(list[key],vals...)
-valueat(x::Number, t) = x
-valueat(f::Function, t) = f(t)
+valueat(x::Number, u, t) = x
+valueat(f::Function, u, t) = f(u,t)
```

Comment:
Fixed with latest commit using a simple try catch block for backwards compatibility. Maybe in the future it will make sense to move to something more general like this.